### PR TITLE
Additional Fields for TPW Hire 

### PR DESCRIPTION
**Pull request by Charlie-Henry**
https://github.com/cityofaustin/atd-data-tech/issues/15519#issuecomment-2086445305

Adding some additional fields for HR Manager->TPW Hire service

### CODE REVIEW

File: services/records_to_knack.py [hunk 1]
```diff
@@ -160,7 +160,11 @@ def main():
     # identify new/changed records and map to destination Knack app schema
     todos = handle_records(data_src, data_dest, field_map, app_name_dest)
 
-    logger.info(f"Updating/creating {len(todos)} records in the destination app.")
+    updates = sum([bool(rec.get("id")) for rec in todos])
+    creates = len(todos) - updates
+
+    logger.info(f"Updating {updates} records in the destination app.")
+    logger.info(f"Creating {creates} records in the destination app.")
```
Comment (frankhereford) [inline services/records_to_knack.py:167]: I like how you broke these out. 💯

**Review by frankhereford — APPROVED** [review]
Looks good Charlie, thank you!

File: services/records_to_knack.py (hunk 1 above)
Comment (johnclary) [reply services/records_to_knack.py:167]: ditto!

**Review by johnclary — APPROVED** [review]
LGTM 🚢 🙏

**Pull request merged**